composer.json

```diff
@@ -47,5 +50,11 @@
             "Laminas\\Server\\": "tests/Stubbed/Laminas/Server"
         }
     },
-    "minimum-stability": "dev"
+    "minimum-stability": "dev",
+    "extra": {
+        "thanks": {
+            "name": "ocramius/proxy-manager",
+            "url": "https://github.com/Ocramius/ProxyManager"
+        }
+    }
```

Comment:
Kudos for adding this little snippet of configuration.

For anyone else that doesn't know that it does: https://github.com/symfony/thanks#forwarding-stars